exercises/space-age/tests/space-age.rs

```diff
@@ -0,0 +1,69 @@
+extern crate space_age;
+
+use space_age::*;
+
+fn assert_in_delta(expected: f64, actual: f64) {
+    let diff: f64 = expected - actual.abs();
+    let delta: f64 = 0.01;
+    if diff > delta {
+        panic!("Your result of {} should be within {} of the expected result {}",
+               actual,
+               delta,
+               expected)
+    }
+}
+
+#[test]
+fn earth_age() {
+    let duration = Duration::from(1_000_000_000);
+    assert_in_delta(31.69, Earth.years_during(&duration));
```

Comment:
This allows for the "unusual" solution of https://gist.github.com/petertseng/ec3887c899f0259661847e8fbfba820d